src/mjolnir/shortcutbuilder.cc

```diff
@@ -246,9 +224,9 @@ bool CanContract(GraphReader& reader,
 
   // Get the directed edges - these are the outbound edges from the node.
   // Get the opposing directed edges - these are the inbound edges to the node.
-  const DirectedEdge* edge1 = tile->directededge(edges[match.first]);
+  const DirectedEdge* edge1 = tile->directededge(edges[0]);
```

Comment:
if you move this and the one below it up above the `EdgesMatch` then you can use it there as well.

Reply:
Oh, got it. That's what you were talking about yesterday